### DISCUSSION TITLE
Release of new version for this library

### DISCUSSION THREAD

**Discussion by antoineKorbit** [PyGithub/PyGithub#2559] [General]
Hello,
Recently, this [PR#2540](https://github.com/PyGithub/PyGithub/pull/2540) has been merged.

I was then wondering when a new version of the PyGithub will be released ? The last one was a more than one month ago. 

Do you have a plan for release or procedure ? I can't find anything about that either in the documentation or issues/discussions in github.

Thanks for the awesome library !

**Comment by EnricoMi** [comment]
The next release is imminent.

**Reply by antoineKorbit** [reply]
Sound good, thanks.

In that case, please see the following issue when you have time:
- https://github.com/PyGithub/PyGithub/issues/2560#issue-1765534373

I hope it's me who doesn't understand how to use the library properly.

**Comment by EnricoMi** [comment]
This has been released.

**Reply by antoineKorbit** [reply]
Great news, thank you sir for the great project. I will try to be more active on this repository starting from now. Hope I can help at some level. Have a good day!

**Reply by EnricoMi** [reply]
@antoineKorbit please watch this project (at least issues and discussions), if you like to answer questions:
![image](https://github.com/PyGithub/PyGithub/assets/44700269/0b53151a-f199-4c4b-83f1-910f5133d714)

Or look out for hot issues and sketch some pull request:

GitHub UI allows to sort for :+1:s:
open :+1: https://github.com/PyGithub/PyGithub/pulls?q=is%3Apr+is%3Aopen+sort%3Areactions-%2B1-desc
closed stale :+1: https://github.com/PyGithub/PyGithub/pulls?q=is%3Apr+sort%3Areactions-%2B1-desc+is%3Aclosed+label%3Astale

Some people love to use :heart::
open :heart: https://github.com/PyGithub/PyGithub/pulls?q=is%3Apr+sort%3Areactions-heart-desc+is%3Aopen+
closed stale :heart: https://github.com/PyGithub/PyGithub/pulls?q=is%3Apr+sort%3Areactions-heart-desc+is%3Aclosed+label%3Astale
